.npmignore

```diff
@@ -47,9 +47,12 @@ src/*
 src/package.json
 src/app/App_Resources
 src/app/app.*
+src/app/main.*
 src/app/assets
 src/app/package.json
-!src/app/services/*
+!src/app/core/*
+!src/app/ui-bootstrap/*
```

Comment:
should you be doing !src/app/ui-bootstrap/*/** ? Because I added the github repo in my nativescript project and installed, and did not see the folders inside core and ui-bootstrap